.github/workflows/js.yml

```
@@ -46,6 +46,7 @@ jobs:
 
       - name: Run Coveralls
         uses: coverallsapp/github-action@v2
+        if: github.repository == 'twbs/bootstrap'
```

Maybe both are working but I've seen and used another approach to do that:
```suggestion
        if: github.repository_owner == 'twbs'
```

---

I think we could probably just use this too:

```
if: ${{ !github.event.repository.fork }}
```